mangaki/mangaki/utils/lasso.py

```diff
@@ -0,0 +1,63 @@
+from mangaki.utils.common import RecommendationAlgorithm
+from mangaki.settings import DATA_DIR
```

Comment:
I'd prefer we use `django` settings, Mangaki's settings can be modified through env variables.